src/userDashboard/dataFormat/CommonDataFormat.js

```diff
@@ -0,0 +1,47 @@
+import { getYear, getMonth } from 'date-fns';
+
+function calculatePerMonth(mapData){
+
+    var result = new Map();

```

Comment:
what is result? 
can you explain it? 
maybe with an example? 
how should the data look in your result var at the end?

Reply:
done.